pkg/ebpf/c/maps.h

```diff
@@ -117,6 +117,7 @@ BPF_HASH(logs_count, bpf_log_t, bpf_log_count_t, 4096);            // logs count
 BPF_PERCPU_ARRAY(scratch_map, scratch_t, 1);                       // scratch space to avoid allocating stuff on the stack
 BPF_LRU_HASH(file_modification_map, file_mod_key_t, int, 10240);   // hold file data to decide if should submit file modification event
 BPF_LRU_HASH(io_file_path_cache_map, file_id_t, path_buf_t, 5);    // store cache for IO operations path
+BPF_LRU_HASH(elf_files_map, file_id_t, bool, 16);                  // store cache for file ELF type check
```

Comment:
Are 16 entries enough here?

Reply:
So the cache here should be for a repetitive write/read probe trigger because of a large file write/read.
The goal here is not to cache it for the case that someone tries to bypass our check (he could then write the first bytes only in the end, and bypass us anyways).